api/tests/unit/infrastructure/externals/pole-emploi/PoleEmploiPayload_test.js

```diff
@@ -31,6 +31,7 @@ describe('Unit | Infrastructure | Externals | Pole-Emploi | PoleEmploiPayload',
       expect(payload.individu).to.deep.equal({
         nom: user.lastName,
         prenom: user.firstName,
+        idPoleEmploi: participation.participantExternalId,
```

Comment:
en vrai c'est pas cet identifiant là, mais le `externalIdentifier`dans `authentification-methods`.

Reply:
**Suggestion:** Je propose de mettre une valeur fictive pour le moment (par exemple: `'NOT-IMPLEMENTED'`) car il s'agit de la documentation et non de l'implémentation du endpoint.

Reply:
Comme il s'agit d'un ticket sur la documentation uniquement, je vais revenir en arrière et ne pas modifier le `PoleEmploiPayload` pour ne pas présager de l'implémentation.